### PR TITLE
Update readme with recent development and Jetbrains as supporter

### PR DESCRIPTION
**Pull request by Harkishen-Singh**
Fixes: #293

### CODE REVIEW

**Comment by Harkishen-Singh** [comment]
@muskankhedia please let me know when this is ready for review.

**Comment by muskankhedia** [comment]
@Harkishen-Singh Yes, it's ready for review.

File: README.md [hunk 1]
```diff
@@ -78,3 +65,7 @@ For more information, regarding usage in different languages. Visit [Bench-Route
 
 - Instant messaging: [bench-routes/community](https://gitter.im/bench-routes/community)
 - Discussions (Recent): https://groups.google.com/forum/#!forum/bench-routes-discussion
+   
+### Supported by
+
+![JetBrains](https://user-images.githubusercontent.com/36980003/111029750-5b46fc80-8424-11eb-800f-ad7f64844f0d.png)
```
Comment (Harkishen-Singh) [inline README.md:71]: This is perfect. Just last one thing, it should be wrapped with a URL that points to jetbrains (use this link: https://jb.gg/OpenSource).

**Review by Harkishen-Singh — COMMENTED** [review]
Sorry, just one last thing before the merge.

**Review by muskankhedia — APPROVED** [review]

**Pull request merged**